ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -100,6 +101,7 @@ export const NetworkListMenu = ({ onClose }) => {
   const selectedTabOrigin = useSelector(getOriginOfCurrentTab);
   const useRequestQueue = useSelector(getUseRequestQueue);
   const networkConfigurations = useSelector(getNetworkConfigurations);
+  const domains = useSelector(getAllDomains);
```

Comment:
not sure how important this is, but somewhere else in code i saw setNetworkClientIdForDomain guarded by hasPermissions instead of checking if the origin has a networkClientId set. The controller itself checks hasPermissions.

Reply:
These should always be in sync and perhaps we should change the controller to look at domains state since the state of which permissions are the threshhold may change